Domain.Interfaces/Repositories/IRepository.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System.Collections.Generic;
 using System;
+using RelationApp.Domain.Models;
 
 namespace RelationApp.Domain.Iterfaces
 {
     public interface IRepository<TEntity>
     {
         IEnumerable<TEntity> GetAll();
 
+        IEnumerable<TEntity> GetFiltered(Func<TEntity, bool> predicate);
+
         TEntity GetOne(Guid id);
 
         void Add(TEntity t);
```

Comment:
Add -> Insert

Reply:
Remove -> Delete

Reply:
Exists - is not nessesery, because you have GetById()
But you can add **bool IsAny(Expression<Func<TEntity, bool>> predicate)**